tla/raft_spec/README.md

```diff
@@ -0,0 +1,39 @@
+# TLA+ model of CCF's implementation of Raft
+
+The given model consists of three files:
+
+- `ccfraft.tla` : The core specification that implements the features as seen below.
+- `MCraft.tla` : The model checking implementation for the spec. Sets the constants and can be modified for each run to
+  fine tune the settings.
+- `MCraft.cfg` : The core configuration which invariants are to be checked etc. Usually stays untouched during normal
+  model checking.
+
+To run this spec, run the `MCraft.tla` with the `-deadlock` parameter to disable liveness checking (see explanation
+below): `tlc MCraft.tla -deadlock`. See the Readme in the parent folder for a full explanation on running the model checker.
+
+## Implemented features
+
+| Feature | Description |
+| :------ | :---------- |
+| Signatures | In CCF, only entries signed by the primary can be committed (in contrast to all entries in normal Raft) |
+| Reconfiguration | CCF implements a one-transaction version of reconfiguration for Raft (read the description [here](https://microsoft.github.io/CCF/main/overview/consensus.html)). |
+
+## Checked invariants
+
+| Name | Short description |
```

Comment:
It could be nice to indicate which ones were in the original Raft model and which ones were added by you? Unless they all fall into the latter category, in which case we should probably mention briefly here.

Reply:
Good point, I clarified this